examples/NSIDC/ISat2_from_CMR_to_S3.ipynb

```diff
@@ -0,0 +1,665 @@
+      "['s3://nsidc-cumulus-prod-protected/ATLAS/ATL03/004/2018/12/06/ATL03_20181206213114_10580102_004_01.h5']\n"
+     ]
+    }
+   ],
+   "source": [
+    "# We can list the s3 links but \n",
+    "for g in granules[0:10]:\n",
+    "    print(g.data_links(only_s3=True))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "mathematical-vitamin",
+   "metadata": {},
+   "source": [
+    "We note that our RelatedLinks array now contain links to AWS S3, these are the direct URIs for our data granules in the AWS us-west-2 region. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "covered-handle",
+   "metadata": {},
+   "source": [
+    "## Data Access using AWS S3\n",
+    "\n",
+    "* **IMPORTANT**: This section will only work if this notebook is running on the AWS **us-west-2** zone\n",
+    "\n",
+    "There is more than one way of accessing data on AWS S3, either downloading it to your local machine using the official client library or using a python library. \n",
+    "\n",
+    "\n",
+    "Related links:\n",
+    "* [HDF in the Cloud challenges and solutions for scientific data](http://matthewrocklin.com/blog/work/2018/02/06/hdf-in-the-cloud)\n",
+    "* [Cloud Storage (Amazon S3) HDF5 Connector](https://www.hdfgroup.org/solutions/enterprise-support/cloud-amazon-s3-storage-hdf5-connector/)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "soviet-shannon",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# READ only temporary credentials\n",
+    "import s3fs\n",
+    "import h5py\n",
+    "\n",
+    "s3_cred = CMR_auth.get_s3_credentials()\n",
+    "\n",
+    "s3_fs = s3fs.S3FileSystem(key=s3_cred['accessKeyId'],\n",
+    "                          secret=s3_cred['secretAccessKey'],\n",
+    "                          token=s3_cred['sessionToken'])\n",
+    "\n",
+    "# Now you could grab S3 links to your cloud instance (EC2, Hub etc) using:\n",
+    "# s3_fs.get('s3://SOME_LOCATION/ATL03_20181015124359_02580106_004_01.h5', 'test.h5')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a29b8c4-f876-4d8f-8fc2-3e00fc83cafb",
+   "metadata": {},
+   "source": [
+    "### We now have the propper credentials and file mapper to access the data within AWS us-west-2. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "other-nowhere",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with s3_fs.open('s3://nsidc-cumulus-prod-protected/ATLAS/ATL03/004/2018/10/15/ATL03_20181015124359_02580106_004_01.h5', 'rb') as s3f:\n",
+    "    with h5py.File(s3f, 'r') as f:\n",
+    "        print([key for key in f.keys()])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "imposed-valuation",
+   "metadata": {},
+   "source": [
+    "### Using xarray to open files on S3\n",
+    "\n",
+    "ATL data is complex so xarray doesn't know how to extract the important bits out of it."
```

Comment:
I feel like more explanation is needed here. If xarray can't get the "important bits," why are you showing me code that uses xarray? Why aren't you showing me how to get the "important bits?"